packages/react-component-library/src/components/Autocomplete/Autocomplete.test.tsx

```diff
@@ -80,7 +80,7 @@ describe('Autocomplete', () => {
 
     describe('when the arrow button is clicked', () => {
       beforeEach(() => {
-        userEvent.click(wrapper.getByTestId('select-arrow-button'))
+        return userEvent.click(wrapper.getByTestId('select-arrow-button'))
```

Comment:
I've gone for returning the promise in most cases as we've been doing that with `waitFor()`. But could also use `await` if we wanted.